src/AirtableRecord.php

```diff
@@ -93,21 +95,4 @@ public function getFields()
     {
         return $this->fields;
     }
-
-    public function setFields($fields): self
-    {
-        $this->fields = $fields;
-
-        return $this;
-    }
-
-    public function getId(): string
```

Comment:
Why did we remove those getters ? The whole point of having an AirtableRecord is to have autocompletion and improve DX to retrieve those values imo :x

Reply:
It's dead code, it's used anywhere, I athink like a PC, `it's not used, so it's unnecessary`. 

About DX of AirtableRecord, it's "poor" (we need to specify in phpdoc the return type of `$record->getFields()`, like [here](https://github.com/TBoileau/Airtable-Client-Bundle/blob/test-sf-integration/tests/Unit/AirtableClientTest.php#L65)), Don't you want to set up some Data Mapping ? :p

Reply:
@liorchamla I put the deleted methods back into AirtableRecord ;)

Reply:
@liorchamla the presence of `setFields` method implicitly means that an AirtableRecord is not immutable. In my opinion, the object must be immutable, so I removed the "setFields" method. What do you think?

Reply:
100% ok :) @ismail1432 already pointed this out :)